### PR TITLE
Add dashboards dir with README

### LINKED ISSUE #20: Create a dashboards directory with README
Add a `dashboards` folder for placing all the Grafana dashboard JSON files for each of the monitored application/service. Include a README describing the purpose of these dashboards.

### LINKED ISSUE #953: Create a dashboards directory with README
Add a `dashboards` folder for placing all the Grafana dashboard JSON files for each of the monitored application/service. Include a README describing the purpose of these dashboards.

### PR DESCRIPTION
**Pull request by hemajv**
Adding a dashboards directory for organizing Grafana monitoring dashboards with a README

(This closes operate-first/support#953)

### CODE REVIEW

File: dashboards/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## Grafana Monitoring Dashboards
+
+[Grafana](https://grafana.com/oss/grafana) is a multi-platform open source analytics and interactive visualization tool. It allows you to query, create dashboards, explore and alert on metrics when connected to supported data sources.
```
Comment (anishasthana) [inline dashboards/README.md]: `explore, and alert`

**Review by anishasthana — APPROVED** [review]

File: dashboards/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+For each of the Operate First managed services and applications, we have created example Grafana dashboards which can be used to visualize our [SLI/SLOs](https://github.com/operate-first/SRE/tree/master/sli-slo) defined for each of the applications.
+
+These monitoring dashboards are useful for visualizing the metrics (such as Prometheus metrics) collected from the applications to track and analyze the system's overall health.
+The dashboards can be altered, saved and reused as per your specific monitoring needs. 
+
+You will find the exported JSON file for each of the dashboards in this folder.
```
Comment (HumairAK) [inline dashboards/README.md]: fyi as far as I know we'll be using the dashboard custom resource that's provided by the grafana operator
see [here](https://github.com/integr8ly/grafana-operator/blob/master/documentation/dashboards.md) for more info. I think you can just paste the raw json in the custom resource though.

File: dashboards/README.md (hunk 1 above)
Comment (hemajv) [reply dashboards/README.md]: accepted the `oxford comma` :)

File: dashboards/README.md (hunk 2 above)
Comment (hemajv) [reply dashboards/README.md]: updated :+1:

**Pull request merged**